pkg/osquery/table/user_avatar_darwin.go

```diff
@@ -0,0 +1,93 @@
+package table
+
+/*
+#cgo darwin CFLAGS: -DDARWIN -x objective-c
+#cgo darwin LDFLAGS: -framework Cocoa -framework Collaboration
+#import <Collaboration/Collaboration.h>
+#include <CoreFoundation/CoreFoundation.h>
+void Image(CFDataRef *imageDataRef, char* user) {
```

Comment:
I'm not very familiar with cgo... Why define this function here rather than calling these functions directly in the Go code?